assets/schema_input.json

```diff
@@ -32,6 +32,14 @@
                 "pattern": "^\\S+\\.g(enome)?(\\.)?vcf(\\.gz)?$",
                 "format": "file-path",
                 "exists": true
+            },
+            "familyPheno": {
+                "errorMessage": "Filename of the pedigree file, mandatory for exomiser",
+                "meta": ["familypheno"],
+                "format": "file-path",
+                "pattern": "^\\S*.y(a)?ml$",
```

Comment:
on acceptait les json aussi?

Reply:
Ah oui c'est vrai bon point. Will do.

Reply:
Done

Reply:
Also included a pattern for the analysis files in the parameters schema.